projects/packages/lazy-images/src/lazy-images.php

```diff
@@ -122,6 +122,8 @@ private function __construct() {
 
 		add_filter( 'wp_kses_allowed_html', array( $this, 'allow_lazy_attributes' ) );
 		add_action( 'wp_head', array( $this, 'add_nojs_fallback' ) );
+
+		add_filter( 'wp_img_tag_add_loading_attr', array( $this, 'maybe_skip_core_loading_attribute' ), 10, 2 );
```

Comment:
I wonder if we should filter through something else, like maybe `the_content`, to catch more use-cases such as the images in the Latest Instagram Posts block?

Reply:
Possibly. I would expect this to catch any issues that rely on the core function for adding the loading attribute. But, in looking for `loading="` in the Jetpack repo, I guess that we've manually set this loading attribute in many places instead of calling a core function.

Reply:
We were already hooking on the `the_content` filter. But, we were pretty conservative on re-processing the content. If we'd already processed any of the images in the content, we didn't touch any of the content. So, I made some changes to always process the content and to handle the logic to not reprocess and image on each image.